src/components/loginForm.vue

```diff
@@ -62,6 +62,7 @@ export default {
   },
```

Comment:
Remove emailErrorMessage and passwordErrorMessage

Reply:
Changes added